src/main/java/emissary/core/MobileAgent.java

```diff
@@ -760,7 +765,22 @@ protected void logAgentCompletion(final IBaseDataObject payloadArg) {
         final Logger dest = (isProbe == null) ? logger : probeLogger;
 
         if (dest.isInfoEnabled()) {
-            dest.info(PayloadUtil.getPayloadDisplayString(payloadArg));
+            // grab reduced-transform-itinerary-list if it exists and add lines to reducedTransformItineraryList
+            List<String> reducedTransformItineraryList;
+            try (Stream<String> lines =
+                    Files.lines(Paths.get(System.getenv("PROJECT_BASE") + "/../src/test/resources/reduced-transform-itinerary-list.csv"))) {
```

Comment:
Related to @andrewbp comment. I think it is a little odd to load what is basically a 'configuration' file from the test resources. I don't believe the test/resources directory is available in a normally running emissary. 
So following the standard .cfg file pattern and keeping it in the config directory might be best. If you need a custom .cfg for unit tests and such that should go in test/resources